python/triton/language/core.py

```diff
@@ -1427,13 +1427,16 @@ def view(input, *shape, _builder=None):
         view(x, (32, 32))
         view(x, 32, 32)
     """
+    warn(
+        f"view is deprecated, please use reshape with reorder being true."
+    )
     shape = _shape_check_impl(_unwrap_iterable(shape))
     return semantic.view(input, shape, _builder)
```

Comment:
If view is equivalent to reshape(can_reorder=True), can we delete the `semantic.view` function entirely?

Reply:
Yes. I think we are planning for a bw-breaking PR, so maybe we should do it at that time?

Reply:
I was talking about removing semantic.view (and making this function call semantic.reshape).  That shouldn't be bw-breaking?